packages/hub/src/scope.ts

```diff
@@ -186,12 +186,20 @@ export class Scope implements ScopeInterface {
   /**
    * @inheritDoc
    */
-  public setTransaction(transaction?: string): this {
-    this._transaction = transaction;
+  public setTransactionName(name?: string): this {
+    this._transactionName = name;
     this._notifyScopeListeners();
     return this;
   }
 
+  /**
+   * Can be removed in major version.
```

Comment:
Is there a list of all the things we need to address in the next major version bump somewhere?

Reply:
Everything is marked with `@deprecated` in the code that we need to address.